decidim-admin/app/assets/stylesheets/decidim/admin/modules/_data-picker.scss

```diff
@@ -0,0 +1,111 @@
+.data-picker {
```

Comment:
That would be great, but how do you do that? I searched for an example in the project and didn't find it. Even, I found the decidim/-admin/app/assets/stylesheets/decidim/admin/utils that has a lot of files copied from decidim-core/app/assets/stylesheets/decidim/utils.

Reply:
Mhhhh, maybe @josepjaume can help here?

Reply:
@leio10 the admin interface is borrowing most of the files but we probably shouldn't be doing that anymore. Have you tried just:

```scss
@import "decidim/modules/data_picker"';
```

Thanks!

Reply:
Mmm, it doesn't work. I did a little research and I'm afraid that it is more complicated than that, and probably implies changes in the decidim-core gem. For example, https://github.com/zurb/foundation-rails includes the shaded SCSS files in the vendor folder, I don't know if that is necesary.

Reply:
@leio10 Can you try:

```diff
diff --git a/decidim-admin/app/assets/stylesheets/decidim/admin/_decidim.scss b/decidim-admin/app/assets/stylesheets/decidim/admin/_decidim.scss
index d17d5e76a..eb597239b 100755
--- a/decidim-admin/app/assets/stylesheets/decidim/admin/_decidim.scss
+++ b/decidim-admin/app/assets/stylesheets/decidim/admin/_decidim.scss
@@ -12,4 +12,6 @@
 
 @import "plugins/foundation-datepicker";
 
+@import "decidim/modules/data-picker";
+
 @import "modules/modules";
diff --git a/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_data-picker.scss b/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_data-picker.scss
deleted file mode 100644
index c5cc0e336..000000000
--- a/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_data-picker.scss
+++ /dev/null
@@ -1,143 +0,0 @@
-.data-picker {
-  &.picker-single .picker-values div a, .picker-prompt a {
-    background: white;
-    cursor: pointer;
-    display: block;
-    width: 100%;
-    margin-bottom: 1rem;
-    padding: 0.7rem 0.5rem;
-    border: 1px solid #eee;
-    border-radius: 4px;
-    box-shadow: inset 0 1px 2px rgba(26, 24, 29, 0.1);
-    -webkit-box-shadow: inset 0 1px 2px rgba(26, 24, 29, 0.1);
-    color: #1a181d;
-
-    &:active {
-      outline: none;
-      border: 1px solid #1a181d;
-      background-color: #fff;
-      box-shadow: 0 0 5px #adadad;
-      -webkit-box-shadow: 0 0 5px #adadad;
-      transition: border-color 0.25s ease-in-out, -webkit-box-shadow 0.5s;
-      -webkit-transition: border-color 0.25s ease-in-out, -webkit-box-shadow 0.5s;
-    }
-
-    &::after {
-      display: block;
-      width: 0;
-      height: 0;
-      border: inset 5px;
-      content: '';
-      border-right-width: 0;
-      border-color: transparent transparent transparent #1a181d;
-      float: right;
-      margin-top: 7px;
-    }
-  }
-
-  &.picker-single .picker-values:not(:empty) + .picker-prompt {
-    display: none;
-  }
-
-  &.disabled.picker-single .picker-values div a, &.disabled .picker-prompt a {
-    cursor: not-allowed;
-    background: #eee;
-
-    &:active {
-      border: inherit !important;
-      box-shadow: inherit !important;
-      -webkit-box-shadow: inherit !important;
-      transition: none !important;
-      -webkit-transition: none !important;
-    }
-  }
-
-  &.picker-multiple .picker-values div {
-    display: block;
-    margin: 0 0 2px;
-    font-size: 0.9rem;
-
-    input {
-      margin: 2px 8px 8px 2px;
-      vertical-align: top;
-    }
-  }
-}
-
-#data_picker-modal {
-  height: 90%;
-
-  .data_picker-modal-content {
-    display: flex;
-    flex-direction: column;
-    max-height: 100%;
-    min-height: 100%;
-
-    .picker-footer {
-      .buttons {
-        display: flex;
-        justify-content: center;
-        padding: 10px 0px 50px; // bottom padding is needed to avoid hiding buttons on mobiles because of https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser
-      }
-    }
-
-    .picker-content {
-      flex-grow: 10;
-      min-height: 100%;
-      max-height: 100%;
-      overflow: auto;
-
-      .picker-scrollable-content {
-        max-height: 100%;
-        overflow: auto;
-      }
-    }
-  }
-}
-
-.scope-picker {
-  ul {
-    margin-left: 0;
-    li {
-      list-style: none;
-    }
-  }
-
-  &.picker-header {
-    padding: 5px;
-    li:hover {
-      background: #ddd;
-    }
-
-    li ~ li a:before {
-      content: '';
-      width: 10px;
-      height: 2em;
-      float: left;
-    }
-    li:last-child a {
-      color: #333;
-    }
-    a {
-      display: block;
-      padding: 2px;
-    }
-  }
-
-  &.picker-content {
-    .picker-scrollable-content {
-      scroll-snap-type: mandatory;
-
-      a {
-        scroll-snap-coordinate: 0px 0px;
-        display: block;
-        border-top: 1px solid #ddd;
-        line-height: 1.5rem;
-        padding: 5px;
-        &:hover {
-          background: #ddd;
-        }
-      }
-    }
-  }
-}
\ No newline at end of file
diff --git a/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_modules.scss b/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_modules.scss
index 29f6d9fbc..2b9ebc4b3 100644
--- a/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_modules.scss
+++ b/decidim-admin/app/assets/stylesheets/decidim/admin/modules/_modules.scss
@@ -15,9 +15,6 @@
 //Process
 @import "process-header";
 
-//Shared modules
-@import "data-picker";
-
 //Contents
 @import "callouts";
 @import "cards";
diff --git a/decidim-core/lib/decidim/core/engine.rb b/decidim-core/lib/decidim/core/engine.rb
index 206dd1d87..c48dd0c16 100644
--- a/decidim-core/lib/decidim/core/engine.rb
+++ b/decidim-core/lib/decidim/core/engine.rb
@@ -56,6 +56,8 @@ module Decidim
       end
 
       initializer "decidim.assets" do |app|
+        app.config.assets.paths << File.expand_path("../../../app/assets/stylesheets", __dir__)
+
         app.config.assets.precompile += %w(decidim_core_manifest.js)
 
         Decidim.feature_manifests.each do |feature|
```

Reply:
Yeah! Works like a charm :+1:
I'll add it now and I hope that adding a new assets path don't change anything else. I suppose that this is the right way to do it, so we will should start to remove duplicated files, as in utils folder.

Reply:
Yeah, I have the same doubts / expectations / suppositions... Let's hope it doesn't break anything and we don't find any really bad side effects from doing this.

Reply:
I guess that it should affects only to the admin appareance. I browsed it and checked that it seems to be the same. Anyway, I think that maybe @josepjaume has something to say :smiley_cat:

Reply:
I was thinking if it'd be better to go with a more conservative approach of only explicitly adding the specific sheets core wants to make public instead of sharing the whole design?